### ISSUE TITLE
There is not FreshIOC.Container.Register<T>() method

### ISSUE THREAD

**Issue by mpalmer78** [rid00z/FreshMvvm#237]
The readme states [here](https://github.com/rid00z/FreshMvvm#ioc-container-lifetime-registration-options) that I can call something like `FreshIOC.Container.Register<HomePageViewModel>();` however there is no Register method that takes zero arguments. Was this removed from the IOC container? How can I register non-interface based instances?

**Comment by SittenSpynne** [comment]
Yeah, I just noticed this too. This is a very useful feature for an IoC container. Is it incorrectly documented, or is it not supposed to be there?

**Comment by hauteur** [comment]
You can, but you have to pass in an interface (or just the same type if you don't have an interface).

So:
`FreshIOC.Container.Register<IViewModel, HomePageViewModel>();`
or
`FreshIOC.Container.Register<HomePageViewModel, HomePageViewModel>();`

Note that doing it that way registers the class as a singleton by default.  You need to add .AsMultiInstance() to override that.
